netlify.toml

```diff
@@ -1,3 +1 @@
-[build]
-  publish = "docs/_build/html"
-  command = "pip install -r docs-requirements.txt && export PYTHONPATH=$(pwd) && python setup.py docs"
+../../templates/docs/netlify.toml.jj2
```

Comment:
wow ... this aint right .. :P

I believe you are on Windows, and this is a bug in the moban program.

Talk to the moban mentors / developers, so they can create a bug report.

Then you need to undo this change.  They can help you with that too.